hdr/libccc/sys/error.h

```diff
@@ -70,10 +70,6 @@ typedef struct ccchandler
 // TODO document this
 #define PARSE_RETURN(VALUE) \
 	{	if (dest)	*dest = (VALUE);	return (i);	}
-// TODO document this
-#define PRINT_RETURN(VALUE) \
-	{	if (dest)	*dest = (VALUE);	return (i);	}
-
```

Comment:
yeah, we can get rid of this - it seems like the point of this macro was to avoid code repetition, but in general if you wanna avoid repeating code for something like this, then you may as well use the common `goto success;`/`goto failure;` pattern.